packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -9,6 +9,7 @@ import { ShoppingBasket } from 'commerce/data';
 
 export default class Basket extends LightningElement {
     @track products = [];
+    @track loading = true;
```

Comment:
I dont think we need the @track decorator here.

Reply:
You are correct.  Removed.